Orange/widgets/visualize/utils/customizableplot.py

```diff
@@ -28,18 +28,23 @@ def available_font_families() -> List:
     if not QApplication.instance():
         _ = QApplication(sys.argv)
     fonts = QFontDatabase().families()
-    default = fonts.pop(fonts.index(default_font_family()))
-    defaults = [default]
-
-    guessed_name = default.split()[0]
-    i = 0
-    while i < len(fonts):
-        if fonts[i].startswith(guessed_name):
-            defaults.append(fonts.pop(i))
-        else:
-            i += 1
-    return FontList(defaults
-                    + [""]
+
+    top_section = []
+
+    default_family = default_font_family()
+    if default_family in fonts:
```

Comment:
I think the `default_family` should be added to fontList even though not present in `fonts`. If not, the combobox may crash when trying to set its text to something missing in its model().